src/components/InstantSearchSsr.js

```diff
@@ -1,6 +1,5 @@
 import { createInstantSearchComponent } from '../util/createInstantSearchComponent';
-import { isVue3 } from 'vue-demi';
-import { h } from 'vue';
+import { isVue3, h } from 'vue-demi';
```

Comment:
can this h be used in vue 2? if so the conditional on isVue3 can go away

Reply:
No, in case of Vue 2, that `h` is undefined and we need to use the one passed to `render` function.